package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "gpii-devpmt",
+    "productName": "devpmt",
+    "version": "1.0.0",
+    "description": "Developers Preference Testing Tool",
+    "main": "standalone.js",
+    "scripts": {
+        "start": "electron standalone.js",
+        "watch-sass": "gulp",
+        "build-sass": "gulp sass"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/sgithens/gpii-devpmt"
+    },
+    "author": "Steven Githens <sgithens@raisingthefloor.org>",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.gpii.net/"
+    },
+    "homepage": "https://github.com/sgithens/gpii-devpmt",
+    "dependencies": {
+        "electron": "1.4.1",
+        "foundation-sites": "6.3.0",
+        "gpii-express": "1.0.2",
+        "gpii-handlebars": "the-t-in-rtf/gpii-handlebars#GPII-2474",
+        "gulp-autoprefixer": "^3.1.0",
+        "handlebars-helpers": "0.8.0",
+        "infusion": "3.0.0-dev.20170127T130413Z.103de6e",
+        "json5": "0.5.1",
+        "lunr": "2.0.3",
+        "marked": "0.3.6",
+        "pagedown": "1.1.0",
+        "universal": "GPII/universal"
+    },
+    "devDependencies": {
+        "electron-packager": "8.5.2",
+        "eslint": "3.14.1",
+        "eslint-config-fluid": "1.1.0",
+        "fluid-grunt-eslint": "18.1.2",
+        "gpii-webdriver": "1.0.0-dev.20170127T135806Z.aa30519",
+        "grunt": "1.0.1",
+        "grunt-jsonlint": "1.1.0",
+        "grunt-shell": "1.3.0",
+        "gulp": "^3.9.1",
```

Comment:
Why both gulp and grunt? It would be reasonable to centralise on just one.

Reply:
I have a custom foundation theme for the project, and the gulp script is from the foundation documentation for compiling and watching changes to it.  It seemed less error prone to just stick with their documentation on how to build it.  I'll either put a note in the README about this extra script... or maybe figure out how to reproduce it in grunt.